server/openai/generateItenerary.js

```diff
@@ -0,0 +1,56 @@
+const openaiClient = require('./openaiClient')
+
+async function generateItinerary(constraints) {
+  try {
+    const conversation = [
+      {
+        role: 'system',
+        content: `You are an AI that generates travel itineraries. Respond with ONLY JSON that contains the plan for each stage. 
+        ONLY respond with the following format, do not include any descriptions or codeblocks, I should be able to parse the output to a JavaScript Object
+        The response needs to be formatted exactly like the following structure. Add some Food and snack stages as well with an optional: true marker.:
+        '''
+        {
+          plan:[
+            {
+              day: Number
+              bullet_point_summary: String (comma seperated)
```

Comment:
What do you envision this to look like? I'm wondering if "A scenic walk along the seawall followed by scrumptious delicacies from local waters." is nicer than a bullet point list, since if we want a bullet point list we can just use a list of the stages' place from the array below

Reply:
so far from testing it out it returns the names of the places as summary points. For example: Stanley Park, Medina Cafe etc. I can add some text to be more verbose.

Reply:
a bit more verbose / descriptive would be good, thx